src/app/legacy/containers/CpsRecommendations/RecommendationsPromo/index.jsx

```diff
@@ -100,12 +101,14 @@ const StyledHeadline = styled.div`
   align-items: center;
 `;
 
-const RecommendationsPromo = ({ promo, eventTrackingData }) => {
+const RecommendationsPromo = ({ promo, eventTrackingData, isArticle }) => {
   const { script, service } = useContext(ServiceContext);
   const handleClickTracking = useCombinedClickTrackerHandler(eventTrackingData);
 
   const { headline, url, indexImage } = extractPromoData({ promo });
 
+  const StyledPromoWrapper = makeStyledPromoWrapper(isArticle);
```

Comment:
You should be able to pass in `isArticle` into the `<StyledPromoWrapper />` component as a prop and then access it in the above Emotion styled component using `props` or destructed.